src/lib/users/local_password.rb

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2016 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require "yast"
+require "users/ca_password_validator"
+
+module Users
+  # Password for a local user
+  #
+  # This class is used to validate the correctness of a password by checking
+  # some security requirements that the password must fulfill.
+  # The class only checks for non-fatal errors. If a password is considered
+  # invalid, it still can be used if the user decides to ignore the warnings.
+  class LocalPassword
+    Yast.import "UsersSimple"
```

Comment:
same comment as before Yast.import is functionality equal require. Yast.include is functionality similar to ruby include. So I think we should have it in similar place and use it similarly